app/js/components/orderEntry/addForm/AddForm.jsx

```diff
@@ -215,23 +215,30 @@ export class AddForm extends React.Component {
       dispensingUnit,
       duration,
       durationUnit,
+      drugInstructions,
     } = this.state.fields;
 
     if (Object.values(this.state.fieldErrors).includes(true)) return true;
 
-    else if (duration && !durationUnit) return true;
-
-    else if ((this.props.careSetting.display === 'Outpatient') &&
-      !(dose &&
+    else if (this.state.formType === 'Standard Dosage') {
+      if (duration && !durationUnit) return true;
+      else if ((this.props.careSetting.display === 'Outpatient') &&
+      !(
+        dose &&
         dosingUnit &&
         frequency &&
         route &&
         dispensingQuantity &&
-        dispensingUnit)) return true;
-
-    else if ((this.props.careSetting.display === 'Inpatient') &&
-    !(dose && dosingUnit && frequency && route)) return true;
-
+        dispensingUnit
+      )) return true;
+      else if ((this.props.careSetting.display === 'Inpatient') &&
+      !(dose && dosingUnit && frequency && route)) return true;
+    } else if (this.state.formType === 'Free Text') {
+      if ((this.props.careSetting.display === 'Outpatient') &&
+      !(drugInstructions && dispensingQuantity && dispensingUnit)) return true;
+      else if ((this.props.careSetting.display === 'Inpatient') &&
+      !(drugInstructions)) return true;
+    }
     return false;
```

Comment:
Generally by OpenMRS convention we also put brackets around if and else blocks... 

https://stackoverflow.com/questions/2125066/is-it-bad-practice-to-use-an-if-statement-without-brackets

Reply:
@mogoodrich Thank you, will fix that